### ISSUE TITLE
Language Search for Metatags

### ISSUE THREAD

**Issue by elkinc2000** [griff/metaz#68]
Hello,

Can you please add a feature to select the language for metatags search. I have searched some tv series in The TV DB, they have spanish data for metatags, but, MetaZ is only taking metatags for english.

Best Regards.

**Comment by gnodet** [comment]
Yeah, I agree this would be nice to choose the language for a given movie/show when querying metadata.

**Comment by gl40** [comment]
I agree too ! (support of french tags for me) 

regards
